io.github.quantum_mutnauq.fast_reader_gtk.yml

```diff
@@ -0,0 +1,78 @@
+app-id: io.github.quantum_mutnauq.fast_reader_gtk
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: fast-reader
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=wayland
+  
+modules:
+  - name: libconfig
+    buildsystem: simple
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
+      - make
+      - make install
+      - rm -r -f ${FLATPAK_DEST}/share/info/
+    sources:
+      - type: git
+        url: https://github.com/hyperrealm/libconfig.git
+        tag: v1.5
+    post-install:
+      # Manuelle Generierung der libconfig.pc Datei
+      - |
+        echo "Generating libconfig.pc"
```

Comment:
why do you need to do that?

Reply:
I made like that because, in the past, the build would fail. Now, it works.

Reply:
then remove it.